src/templates/head.html

```diff
@@ -28,14 +35,13 @@
   {% if App.Config.configArr.devmode %}
     <script defer src='app/js/vendors-node_modules_fancyapps_fancybox_dist_jquery_fancybox_js-node_modules_fortawesome_font-4ecb99.bundle.js?v={{ v }}'></script>
   {% else %}
-    <script defer src='app/js/636.bundle.js?v={{ v }}'></script>
+    <script defer src='app/js/vendors.bundle.js?v={{ v }}'></script>
```

Comment:
ok now I see what the "vendors" do in webpack config

Reply:
See other comment https://github.com/elabftw/elabftw/pull/2255#discussion_r593766482